test/index.test.js

```diff
@@ -138,13 +138,16 @@ describe('htmlparser2 option', () => {
 });
 
 describe('trim option', () => {
-  it('preserves whitespace text nodes when disabled (default)', () => {
+  it('preserves whitespace text nodes when disabled if valid in parent (default)', () => {
     const html = `<table>
   <tbody>
+    <tr><td>\n</td>\t</tr>\r
   </tbody>
 </table>`;
     const reactElement = parse(html);
-    expect(render(reactElement)).toBe(html);
+    expect(render(reactElement)).toBe(
+      '<table><tbody><tr><td>\n</td></tr></tbody></table>'
+    );
```

Comment:
Can you also add a snapshot test?

```js
expect(reactElement).toMatchInlineSnapshot();
```

I'm curious to see how the snapshot will format the output